pkg/apis/kops/v1alpha1/instancegroup.go

```diff
@@ -89,7 +89,7 @@ type InstanceGroupSpec struct {
 	Kubelet *KubeletConfigSpec `json:"kubelet,omitempty"`
 	// Taints indicates the kubernetes taints for nodes in this group
 	Taints []string `json:"taints,omitempty"`
-	// AdditionalUserData is any aditional user-data to be passed to the host
```

Comment:
Can you check to see if this mistake also exists on: 
https://github.com/kubernetes/kops/blob/master/pkg/apis/kops/v1alpha2/instancegroup.go
https://github.com/kubernetes/kops/blob/master/pkg/apis/kops/instancegroup.go